src/tests/test_command_list.py

```diff
@@ -312,10 +308,12 @@ def test_applications_clone_exceptions(self):
             application_exists_mock.return_value = False, ""
             retrieve_appname_and_path_mock.return_value = self.path, self.application
             # When application is valid (no items in error, warning and info)
-            applications_validate_mock.return_value = {"error": {"an_error"}, "warning": {}, "info": {}}
+            applications_validate_mock.return_value = {"error": [ "an_error" ], "warning": [], "info": []}
```

Comment:
this is not PEP8.  ["an_error"] is PEP8. Also counts for line 424 and the other instances where it is changed